README.md

```diff
@@ -49,6 +49,21 @@ public enum Value {
 }
 ```
 
+### Listen and Notify
+
```

Comment:
This section is a good example of how to get it to work but can you provide some context before the example of why you would use listen and notify?

I'd also recommend linking to the docs for more detail. https://www.postgresql.org/docs/current/static/sql-notify.html